deps/spidershim/test/value.cc

```diff
@@ -778,6 +778,24 @@ TEST(SpiderShim, Date) {
   EXPECT_EQ(time, Date::Cast(*date.ToLocalChecked()->ToObject())->ValueOf());
 }
 
+namespace {
```

Comment:
There are a number of tests about external strings in v8's `test-api.cc`, such as `ScriptUsingStringResource`, `ScriptUsingOneByteStringResource`, and `ScriptMakingExternalString`.  Out of curiosity, have you tried adopting those tests?  Massaging the tests a bit to get them to compile is fine.  And if they have parts that depend on features we don't support yet, you can comment them out too.

(FWIW this can happen in a different PR, not going to block this on porting those tests, but I've been trying to adopt v8 tests where possible, to make sure we agree on the API semantics and whatnot.)